sbncode/FlashMatch/FlashPredict_module.cc

```diff
@@ -351,43 +375,34 @@ void FlashPredict::produce(art::Event& evt)
     FlashMetrics flash = {};
     Score score = {std::numeric_limits<double>::max()};
     bool hits_ophits_concurrence = false;
-    for(auto& simpleFlash : simpleFlashes) {
-      unsigned ophsInVolume = simpleFlash.ophsInVolume;
-      if(hitsInVolume != ophsInVolume){
-        if(fSBND){
-          if(fForceConcurrence) continue;
-          else if((hitsInVolume < kActivityInBoth) &&
-                  (ophsInVolume < kActivityInBoth)) {
-            continue;
+    for(auto& origFlash : flashMetrics) {
+      unsigned ophsInVolume = origFlash.activity;
+      if(fIsSimple) {
+        if(hitsInVolume != ophsInVolume){
+          if(fSBND){
+            if(fForceConcurrence) continue;
+            else if((hitsInVolume < kActivityInBoth) &&
+                    (ophsInVolume < kActivityInBoth)) {
+              continue;
+            }
           }
-        }
-        else if(fICARUS){
-          if((hitsInVolume < kActivityInBoth) &&
-             (ophsInVolume < kActivityInBoth)) {
-            continue;
+          else if(fICARUS){
+            if((hitsInVolume < kActivityInBoth) &&
+               (ophsInVolume < kActivityInBoth)) {
+              continue;
+            }
+            else if(fForceConcurrence && hitsInVolume == kActivityInBoth) continue;
           }
-          else if(fForceConcurrence && hitsInVolume == kActivityInBoth) continue;
         }
-      }
-      hits_ophits_concurrence = true;
+      }  
```

Comment:
It might be a GitHub thing, but it seems like one or a few of these lines have broken indentation.